paymentsheet/res/menu/paymentsheet_payment_methods_list.xml

```diff
@@ -1,7 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Was this unused?

Reply:
Yes!

(Sorry, I think this came from https://github.com/stripe/stripe-android/pull/6603 and I didn’t undo this change when branching off into this pull request.)